src/calculateRentalCost.js

```diff
@@ -3,8 +3,28 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
```

Comment:
Prefer `const` over `let` for `totalCost` since it's not reassigned, only modified within the `if` conditions, which doesn't prevent it from being a `const` - see checklist item 3.